backend/application/syson-application/src/test/java/org/eclipse/syson/application/export/ImportExportTests.java

```diff
@@ -44,11 +45,15 @@ public class ImportExportTests extends AbstractIntegrationTests {
     @Autowired
     private SysMLv2DocumentExporter exporter;
 
+    @Autowired
+    private SysMLEditingContextProcessor sysMLEditingContextProcessor;
+
```

Comment:
Why do you add 2 empty lines? Please just let one.

Reply:
Done.